src/main/scala/mesosphere/marathon/api/akkahttp/v2/GroupsController.scala

```diff
@@ -76,4 +96,29 @@ class GroupsController(electionService: ElectionService, infoService: GroupInfoS
     }
   }
   // format: On
+
+  def authorizationSelectors(implicit identity: Identity): GroupInfoService.Selectors = {
+    GroupInfoService.Selectors(
+      AppHelpers.authzSelector,
+      PodsResource.authzSelector,
+      authzSelector)
+  }
+
+  import mesosphere.marathon.api.v2.InfoEmbedResolver._
+  /**
+    * For backward compatibility, we embed always apps, pods, and groups if nothing is specified.
+    */
+  val defaultEmbeds = Set(EmbedApps, EmbedPods, EmbedGroups)
+  def extractEmbeds: Directive1[(Set[AppInfo.Embed], Set[GroupInfo.Embed])] = {
```

Comment:
Should this be part of the `Directives.scala`?

Reply:
I would prefer not have this here since it is very specific to groups - it has specific defaults for groups and calls different embed resolver than e.g. apps controller